schema.sql

```diff
@@ -9,3 +9,7 @@ CREATE TABLE animals(
     neutered BOOLEAN NOT NULL,
     weight_kg NUMERIC(5,2) NOT NULL
 );
+
+-- QUWERY AND UPDATE ANIMALS TABLE
+ALTER TABLE animals
+ADD COLUMN species TEXT;
```

Comment:
- [ **OPTIONAL** ] As required for the project to add a new column called `species` with a `string` type, I think it would be better if you modify the `TEXT` type to ``varchar`` as this will help you to adjust the length (e.g.: `varchar(255)` in this example) based on your needs.

Reply:
Thank you for the suggestion and the review @tresorsawasawa. Noted.